integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/End2EndTestIT.java

```diff
@@ -190,6 +203,15 @@ public void testAddProcessor() throws Exception {
         assertThat(action.getParameters()).containsEntry(KafkaTopicAction.TOPIC_PARAM, TOPIC_NAME);
     }
 
+    @Test
```

Comment:
All the tests in this class are disabled. Is it possible to move this test to another module?